third_party/gRPC_utils/proto_reflection_descriptor_database.cc

```diff
@@ -315,7 +326,11 @@ void ProtoReflectionDescriptorDatabase::AddFileFromResponse(
 
 const std::shared_ptr<ProtoReflectionDescriptorDatabase::ClientStream>
 ProtoReflectionDescriptorDatabase::GetStream() {
-  if (!stream_) {
+  if (!stream_) //only assign deadline to a unary rpc.
```

Comment:
I think this is not the place you are looking for.
It is kind of a stream singleton constructor. if stream does not exist it is creating it.
reflection is always a stream.